convert2rhel/checks.py

```diff
@@ -791,10 +791,20 @@ def is_loaded_kernel_latest():
         # Repoquery failed to detected any kernel or kernel-core packages in it's repositories
         # we allow the user to provide a environment variable to override the functionality and proceed
         # with the conversion, otherwise, we just throw an critical logging to them.
-        unsupported_skip = os.environ.get("CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK", "0")
-        if unsupported_skip == "1":
+        allow_older_envvar_names = (
+            "CONVERT2RHEL_ALLOW_SKIP_KERNEL_CURRENCY_CHECK",
+            "CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK",
+        )
+        if any(envvar in os.environ for envvar in allow_older_envvar_names):
```

Comment:
@Andrew-ang9 I think it would be nice to add a comment above this variable definition explaining that this is just something we did to allow users in the next versions to have time to adjust their environment variables names, and that it will be removed in the future, as we plan to only move forward with the `CONVERT2RHEL_ALLOW_SKIP_KERNEL_CURRENCY_CHECK`

Reply:
I am writing up that comment and will have it pushed on both this pr and PR#683 as this comment goes for both this and that, thank you for suggesting this!